### DISCUSSION TITLE
Wildcard route blocks static file access

### DISCUSSION THREAD

**Discussion by guoyunhe** [adonisjs/core#4192] [Help] [answered]
In my Adonis app, I use React as front-end framework and handling page routing at front-end. In Adonis' routes, I use:

```ts
Route.group(() => {
  // API routes...
}).prefix('/api');

// Web page routes
Route.get('*', async ({ view }) => {
  const html = await view.render('app');
  return html;
});
```

However, this wildcard route blocks my access to static files in `/public`. (node ace serve --watch) Images like `/logo.svg` won't load.

Repo https://github.com/guoyunhe/yara

**Comment by RomainLanz** [comment, marked as answer]
You miss the configuration for static assets.

https://docs.adonisjs.com/guides/static-assets

**Reply by guoyunhe** [reply]
Yeah, it worked. Maybe it is because I created this project with `api` mode and then I start to add assets stuff back...

**Reply by RomainLanz** [reply]
Yup, API preset does not include handling of static assets
